arch/risc-v/src/mpfs/mpfs_spi.h

```diff
@@ -75,6 +75,32 @@ struct spi_dev_s *mpfs_spibus_initialize(int port);
 
 int mpfs_spibus_uninitialize(struct spi_dev_s *dev);
 
+/****************************************************************************
+ * Name:  mpfs_spi0/1/..._select
+ *
+ * Description:
+ *   The external function, mpfs_spi0/1/..._select,
+ *   must be provided by board-specific logic.  This is implementation of the
+ *   select of the SPI interface defined by
+ *   struct spi_ops_s (see include/nuttx/spi/spi.h).
+ *   All other methods (including mpfs_spibus_initialize())
+ *   are provided by common MPFS logic.
+ ****************************************************************************/
+
+#ifndef CONFIG_SPI_CS_CONTROL
+
+#ifdef CONFIG_MPFS_SPI0
+void mpfs_spi0_select(FAR struct spi_dev_s *dev,
+                        uint32_t devid, bool selected);
+#endif
+
+#ifdef CONFIG_MPFS_SPI1
+void mpfs_spi1_select(FAR struct spi_dev_s *dev,
+                        uint32_t devid, bool selected);
+#endif
+
```

Comment:
Should mpfs_spi0_select() and mpfs_spi1_select() have weak_function counterparts? So it would compile if the configuration is used? Or does it matter?

Reply:
I will add that still, thanks!